File: examples/codegen-result/src/typescript/api/resp/PageInfo.ts

/**
 * 分页对象
**/

export interface  PageInfo<T> {

        /**
            *属性说明：查询页码，示例输入：
            *在java中的类型为：Integer
        **/
        queryPage?: number;
        /**
            *属性说明：查询大小，示例输入：
            *在java中的类型为：Integer
        **/
        querySize?: number;
        /**
            *属性说明：响应集合列表，示例输入：
            *在java中的类型为：List
        **/
        records?: Array<T>;
}

File: examples/swagger-2/src/main/java/com/wuxp/codegen/swagger2/example/resp/PageInfo.java
package com.wuxp.codegen.swagger2.example.resp;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

import java.util.List;

@ApiModel("分页对象")
public class PageInfo<T> {

    @ApiModelProperty("响应集合列表")
    List<T> records;

    @ApiModelProperty("查询页码")
    Integer queryPage;

    @ApiModelProperty("查询大小")
    Integer querySize;

    public List<T> getRecords() {
        return records;
    }
}
